<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Normeform</class>
 <widget class="QDialog" name="Normeform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>995</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>1820</x>
      <y>0</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/logo.jpg);</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>161</width>
     <height>995</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));
border-top-right-radius: 50px;
color: white;
font: 75 20pt &quot;Berlin Sans FB&quot;;</string>
   </property>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);
border-radius: 10px;
color: white;</string>
    </property>
    <property name="text">
     <string>Auditeur</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);
border-radius: 10px;
color: white;</string>
    </property>
    <property name="text">
     <string>Client</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);
border-radius: 10px;
color: white;</string>
    </property>
    <property name="text">
     <string>Rapport</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);
border-radius: 10px;
color: white;</string>
    </property>
    <property name="text">
     <string>Materielle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);
border-radius: 10px;
color: white;</string>
    </property>
    <property name="text">
     <string>Norme</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>89</y>
     <width>1759</width>
     <height>904</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Times New Roman&quot;;</string>
   </property>
   <widget class="QPushButton" name="pushButton_todolist">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>To Do List</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_description">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>391</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gestions des normes</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_decroissant">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>330</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tri décroissant</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Etat</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>290</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Numéro du norme</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_chatbox">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>650</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Chatbox</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ajouter">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>410</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_supprimer">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>290</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_video">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>460</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Video</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_numero">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>420</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rechercher">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>290</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Rechercher</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Titre</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_croissant">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>330</y>
      <width>97</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Tri croissant</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_confirme">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>97</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirmé</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_modifier">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>420</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Modifier</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_refuse">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>320</y>
      <width>97</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Refusé</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_statistique">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>580</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Statistique</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>370</y>
      <width>601</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_PDF">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>520</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>PDF</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_titre">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_supprimer">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>290</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Supprimer</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
